tasks/main.yml

```diff
@@ -197,6 +197,20 @@
   changed_when: not __realm_join_output.stderr is
     search("Already joined to this domain")
 
+- name: Configure SSSD settings
+  community.general.ini_file:
+    path: /etc/sssd/sssd.conf
+    state: "{{ item.state | default('present') }}"
+    section: "sssd"
+    option: "{{ item.key }}"
+    value: "{{ item.value }}"
+    create: true
+    owner: root
+    group: root
+    mode: u=rw,g=,o=
+  loop: "{{ ad_integration_sssd_settings }}"
+  notify: Handler for ad_integration to restart services - sssd
+
```

Comment:
Do any of these sssd settings need to be set, and sssd restarted, before trying to join to the AD domain?  The task `name: Run realm join command`?  Do any of these sssd settings also require realmd to be restarted, or otherwise notified (e.g. like a `SIGHUP`)?  @jakub-vavra-cz @justin-stephenson wdyt?

Reply:
>> Do any of these sssd settings need to be set, and sssd restarted, before trying to join to the AD domain?

No, SSSD only gets involved after the system is joined to the domain. `realm join` creates a sssd.conf, so SSSD config changes should be made only after the realm join is performed or they will likely be overwritten.

realmd restart is not needed, SSSD restart is enough.